### ISSUE TITLE
This app is abandoned

### ISSUE THREAD

**Issue by foxjaw** [SubhamTyagi/AppLock#64]
Don't use this app, as the developer gone offline ages ago & stopped maintaining his projects.

**Comment by anonfaded** [comment]
> Don't use this app, as the developer gone offline ages ago & stopped maintaining his projects.

But isn't the app working? Or is there any active adless applock?

**Comment by anonfaded** [comment]
oh thats weird, I just came to know that in android 11+ versions google made a restriction in which we can't access the android/data and android/obb  directories and now this applock :confused: 
Btw i am on android 11 so is there any good applock that has atleast less ads than the others ? i tried this one and it sometimes work but after closing apps from recent apps, the applock stops working...

**Comment by anonfaded** [comment]
yeah its old but the latest android version is very strict which i don't like :')

**Comment by anonfaded** [comment]
i was trying to do a custom rom but i just realized that there isn't a single custom rom  for my samsung haha

**Comment by anonfaded** [comment]
hmm i can use applocks in android 11 its working but the ads are just tooo much, but there isn't any way so i have to continue it this way haha

**Comment by SubhamTyagi** [comment]
The restriction introduced since release of Android 10 the app is not working as intended. Due to no free time this app is not maintained and will not received any updates
